src/Microsoft.SqlTools.ServiceLayer/Connection/ReliableConnection/ReliableSqlConnection.cs

```diff
@@ -434,12 +434,12 @@ private void RetrieveSessionId()
                         if (DBNull.Value != result)
                         {
                             string sessionId = (string)command.ExecuteScalar();
-                            _azureSessionId = new Guid(sessionId);
+                            Guid.TryParse(sessionId, out _azureSessionId);
```

Comment:
should we add an error or warning when the parsing fails here?

Reply:
Good point - it previously would fall through to the catch statement & log (for SQLExceptions, not the FormatException causing this issue). I'll add logging